### ISSUE TITLE
JAWS screen reader does not read Date picker and time picker distinctively

### ISSUE THREAD

**Issue by AnushkaAG29** [SAP/fundamental-ngx#12782]
#### Is this a bug, enhancement, or feature request?
Feature request

#### Describe your proposal.
There is no distinction between the date picker and the time picker which is essential for accessibility.

#### Can you handle that on the application side
--

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
0.53.1

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
The JAWS screen reader reads only picker, instead, it should read date picker or time picker distinctively.

#### Please provide relevant source code (if applicable).
https://sap.github.io/fundamental-ngx/#/platform/date-picker
https://sap.github.io/fundamental-ngx/#/platform/datetime-picker

#### Please provide stackblitz example(s).
--

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, consulted with an accessibility expert. Date and time pickers should be read out as is.

#### Did you check the documentation and the API?
Checked the documentation and API.

#### Did you search for similar issues?
--

#### Is there anything else we should know?
--

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@nikolay-kolarov Do we need to announce date picker or time picker instead?

**Comment by AnushkaAG29** [comment]
Yes, the picker should specify whether it is a date picker or time picker.

**Comment by nikolay-kolarov** [comment]
Hi,

You can use "Edit Date Picker" and "Edit Date Time Picker", but please double check the exact text with UX.

Best Regards,
Nikolay
